src/Raven.Studio/wwwroot/Content/css/bootstrap/tooltip.less

```diff
@@ -8,6 +8,7 @@
   position: absolute;
   z-index: @zindex-tooltip;
   display: block;
+  pointer-events: none;
```

Comment:
can you check if it won't break tooltips with hover? Sometimes we have copy buttons etc inside tooltips.

Reply:
That's right, when you hovered over the tooltip it disappears. I have rolled back this change